src/dropdown/dropdown.scss

```diff
@@ -203,7 +203,10 @@ $height-small: 30px !default;
 
   &:not(.is-disabled):hover {
     background: $background-hover;
-    color: $text-color-hover;
+
+    &.has-value .Select-value .Select-value-label {
+      color: $text-color-hover;
```

Comment:
wait, this is really different from before: you're using `$text-color-hover` only when it has a value? why?